src/App.js

```diff
@@ -1,22 +1,26 @@
-/* eslint-disable import/no-extraneous-dependencies */
 import './App.css';
 import { Route, Routes } from 'react-router-dom';
-import { Provider } from 'react-redux';
+// import { useEffect } from 'react';
+// import { useDispatch } from 'react-redux';
 import NavBar from './components/NavBar';
 import Home from './pages/Home';
 import Categories from './pages/Categories';
-import store from './redux/store';
+// import { getBooks } from './redux/books/booksSlice';
 
 function App() {
+  // const { bookItem } = useSelector((store) => store.book);
+  // const dispatch = useDispatch();
+  // useEffect(() => {
+  //   dispatch(getBooks());
+  // }, []);
+
```

Comment:
- [OPTIONAL] It would be better to delete the commented line of code to have a clean codebase 🙃

Reply:
okay, I will thank you